lib/_stream_readable.js

```diff
@@ -1081,6 +1081,16 @@ Object.defineProperty(Readable.prototype, 'readableLength', {
   }
 });
 
+Object.defineProperty(Readable.prototype, 'readableObjectMode', {
+  enumerable: false,
+  get() {
+    return this._readableState && this._readableState.objectMode;
+  },
+  set(value) {
+    this._readableState && (this._readableState.objectMode = value);
```

Comment:
Is `objectMode` ever changed during the lifetime of a stream? If not do we need the setter?

Reply:
You are right. We do not support changing this at runtime.